src/main/resources/io/jenkins/plugins/oidc_provider/config/IdTokenConfiguration/config.jelly

```diff
@@ -28,31 +28,31 @@ THE SOFTWARE.
     <f:section title="OpenID Connect">
         <f:advanced title="${%Claim templates}" align="left">
             <f:entry field="claimTemplates" title="${%General claim templates}">
-                <f:repeatableProperty field="claimTemplates">
+                <f:repeatableHeteroProperty field="claimTemplates" honorOrder="true" hasHeader="true">
```

Comment:
Why? This is not a hetero field—the elements are all of the same type, `ClaimTemplate`.

Reply:
<img width="481" alt="Screenshot 2022-11-19 at 13 15 46" src="https://user-images.githubusercontent.com/3195424/202852647-1a2c0212-7e92-4055-b998-f92a09908631.png">

It was the only way I was able to get the moveable ordering, is there a better way todo this?

Reply:
Pretty sure `f:repeatableProperty` has some similar option. Whatever the case, `f:repeatableHeteroProperty` is not what you want.

Reply:
The `honorOrder` attribute only appears to be supported for `hetero-list` or `repeatableHeteroProperty`
https://reports.jenkins.io/core-taglib/jelly-taglib-ref.html#repeatableheteroproperty

I don't see anything on `repeatableProperty` that would do this?
https://reports.jenkins.io/core-taglib/jelly-taglib-ref.html#repeatableproperty

Reply:
> `header`
>  For each item, add this header. This also activates drag&drop (where the header is a grip).